src/webui/www/public/setlocation.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>QBT_TR(Set location)QBT_TR[CONTEXT=TransferListWidget]</title>
```

Comment:
>Should new labels not be added because they're not yet translated?

Yes, otherwise there is no point of doing so...
we have src/webui/extra_translations.h , I think it's possible to put it there.